src/tab.js

```diff
@@ -160,18 +195,6 @@
       settings,
     });
 
-    // https://gjs-docs.gnome.org/webkit240~4.0_api/webkit2.webinspector
-    // const webInspector = webView.get_inspector();
-    // webView.connect("load-changed", (self, load_event) => {
-    //   webInspector.show();
-    // });
-
-    // webView.connect("decide-policy", (self, decision, decision_policy) => {
-    //   if (decision_policy === PolicyDecisionType.NEW_WINDOW_ACTION)
-    //     log(decision);
-    //   log(decision_policy);
-    // });
```

Comment:
please don't remove original comments

Reply:
Yeah, sure I am sorry. I got confused as I did't find this one (I think)

> `         //   git ("permission request"); `
and also didn't remember adding it, so I removed the whole block I guess.